<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VSSim</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="topMargin">
          <number>10</number>
         </property>
         <item row="0" column="1">
          <widget class="QSpinBox" name="serviceUnitsCount">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="specialValueText">
            <string>oo</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Number of service units</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="incomingRate">
           <property name="placeholderText">
            <string>Enter value</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="serviceRate">
           <property name="placeholderText">
            <string>Enter value</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Average distance of incoming requests</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Average time for requests to be finished</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="enableMeasureEvents">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Enable measure events</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Distance between measure events</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="measureEventDistance">
           <property name="text">
            <string>50</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Precision for stop check (1.0e-X)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="precision">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startSimulationButton">
         <property name="text">
          <string>Start simulation</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="verticalGroupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Results</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="2" column="1">
          <widget class="QLineEdit" name="valueNQ">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;T&lt;span style=&quot; vertical-align:sub;&quot;&gt;Q&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>N</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Standard derivation of N</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Standard derivation of T</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QProgressBar" name="standardDerivationT">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QProgressBar" name="standardDerivationN">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>T</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;N&lt;span style=&quot; vertical-align:sub;&quot;&gt;Q&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="valueTQ">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Simulation time</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="simTime">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="valueN">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Standard derivation of T&lt;span style=&quot; vertical-align:sub;&quot;&gt;Q&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QProgressBar" name="standardDerivationTQ">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="valueT">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Standard derivation of N&lt;span style=&quot; vertical-align:sub;&quot;&gt;Q&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QProgressBar" name="standardDerivationNQ">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="nCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QCheckBox" name="tCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QCheckBox" name="nqCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QCheckBox" name="tqCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Finished</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>startSimulationButton</tabstop>
  <tabstop>valueT</tabstop>
  <tabstop>valueNQ</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
